en/overrides.md

```diff
@@ -30,6 +30,46 @@ To remove all current overrides on a channel:
 
     sudo snap-proxy delete-override <snap> <channel>
 
+### Revisions and Architectures
+
+Every *.snap file has a unique
+[revision](https://snapcraft.io/docs/getting-started) within a channel.
+
+One revision can support one or multiple architectures. Specifying a revision
+for an override will therefore determine for which architectures the override is
+created.
+
+Revisions for specific snaps can be looked up using the `snap info` command,
+which will list currently available revisions for the architecture of the device
+running this command. Snap Stores Devices API
+[snaps_info](https://api.snapcraft.io/docs/info.html) endpoint can also be used
+to obtain available revisions for selected architectures.
+
+In the example below, we have the `core18` snap and two revisions, each
+supporting one architecture.
+
+```
+# 1722 is one of the amd64 revisions of the core18 snap.
+$ sudo snap-proxy override core18 stable=1722
+core18 stable amd64 1722
+
+# 1725 is one of the armhf revisions of the core18 snap.
+$ sudo snap-proxy override core18 stable=1725
+core18 stable armhf 1725
+
+# We can see that we've overriden the stable channel revisions for both
+# amd64 and armhf and that both upstream counterparts ar at lower revisions.
+$ sudo snap-store-proxy list-overrides core18
+core18 stable amd64 1722 (upstream 1705)
+core18 stable armhf 1725 (upstream 1706)
+
+# Deleting a channel specific override will delete overrides for all revisions
```

Comment:
* "channel-specific override" (you need the hyphen)
* don't use the future tense ("will")
So: "Deleting a channel-specific override deletes overrides for all revisions and architectures.